spring-cloud-kubernetes-test-support/src/main/java/org/springframework/cloud/kubernetes/integration/tests/commons/native_client/Util.java

```diff
@@ -298,6 +300,49 @@ public void setUp(String namespace) {
 
 	}
 
+	public void setUpClusterWideClusterRoleBinding(String serviceAccountNamespace) {
+
```

Comment:
needed because k8s-native client requires cluster-wide settings when you ask for "all namespaces"

Reply:
Is this documented?

Reply:
you mean in our documentation? If so, no, not really. We mention a minor note in the property sources part of the documentation, but not in the discovery one. You think such should be added?

Reply:
Yes I do, that can be a separate PR though

Reply:
OK, will add it in a separate PR. thank you.